sdk/cosmos/azure_data_cosmos/src/options/delete_container_options.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#[cfg(doc)]
+use crate::clients::ContainerClientMethods;
+
+/// Options to be passed to [`ContainerClient::delete()`](crate::clients::ContainerClient::delete()).
+#[derive(Clone, Debug, Default)]
+pub struct DeleteContainerOptions {}
+
+impl DeleteContainerOptions {
+    /// Deletes a new [`DeleteContainerOptionsBuilder`](DeleteContainerOptionsBuilder) that can be used to construct a [`DeleteContainerOptions`].
```

Comment:
A little overzealous find and replace here?

Reply:
```suggestion
    /// Creates a new [`DeleteContainerOptionsBuilder`](DeleteContainerOptionsBuilder) that can be used to construct a [`DeleteContainerOptions`].
```

Reply:
Yep. Definitely looking forward to having some macros or something to reduce duplication here 😅